api/envoy/api/v3alpha/core/base.proto

```diff
@@ -150,6 +150,17 @@ message RuntimeUInt32 {
   string runtime_key = 3 [(validate.rules).string = {min_bytes: 1}];
 }
 
+// Runtime derived bool with a default when not specified.
+message RuntimeFeatureFlag {
+  // Default value if runtime value is not available.
```

Comment:
Presumably the default is true since this is a WKT vs. bool? Can you clarify in the docs?